integration_tests/publish_intent_requests_test.go

```diff
@@ -110,6 +110,21 @@ var _ = Describe("Publish Intent Requests", func() {
 					expectedResponse = HTTPTestResponse{Code: 409, Body: urlArbiterResponseBody}
 					assertSameResponse(actualResponse, &expectedResponse)
 				})
+
+				It("returns a 500 error with a custom response", func() {
+					urlArbiterResponseCode = 418
+					urlArbiterResponseBody = `{ "message": "I'm a teapot"}`
+
+					actualResponse := doRequest("PUT", endpoint, publishIntentPayload)
+
+					Expect(testURLArbiter.ReceivedRequests()).To(HaveLen(1))
+					Expect(testDraftContentStore.ReceivedRequests()).To(BeEmpty())
+					Expect(testLiveContentStore.ReceivedRequests()).To(BeEmpty())
+
+					expectedResponseBody := `{"message": "Unexpected error whilst registering with url-arbiter"}`
+					expectedResponse = HTTPTestResponse{Code: 500, Body: expectedResponseBody}
+					assertSameResponse(actualResponse, &expectedResponse)
+				})
```

Comment:
I think you're right - it is worth exploring in a separate PR. One option is to have some simple integration tests and more detailed unit tests of the url-arbiter client.